handout/modules/lazy-load-di.md

```diff
@@ -0,0 +1,93 @@
+# Lazy Loading and the Dependency Injection Tree
+
+Lazy loaded modules create their own branch on the Dependency Injection (DI) tree. This means that it's possible to have services that belong to a lazy loaded module, that are not accessible by the root module or any other eagerly loaded module of our application.
```

Comment:
Yes, this is covered in the section "Sharing the Same Dependency Injection Tree"